lib/external-housing-available/external-housing-available.models.js

```diff
@@ -38,11 +39,24 @@ module.exports = {
     AuthorId: Joi.string().guid(),
     title: Joi.string(),
     address: Joi.string(),
-    beds: Joi.number().integer().min(0).max(10),
-    baths: Joi.number().precision(2).min(0).max(9),
+    bedrooms: Joi.number().integer().min(0).max(10),
+    bathrooms: Joi.number().precision(2).min(0).max(9),
     contact: Joi.string(),
     body: Joi.string(),
     status: Joi.string().valid('active', 'pending', 'inactive'),
-    url: Joi.string().required().uri({scheme: ['https','http']})
+    url: Joi.string().required().uri({scheme: ['https','http']}),
+    price: Joi.number().precision(2).greater(100.00),
+  }),
+  apiFilterQuery: Joi.object().keys({
+    title: Joi.string().optional(),
```

Comment:
I can't remember why I have optional on all these fields in property. I wonder if it's because we have a more complex statement that uses the .without method. not sure. You don't _need_ to do anything since I presume this works here jsut fine as well, it just may not be needed.

Reply:
Yes, I think in property there was a "search" query therefore it is more complex.